packages/cf-component-tabs/example/basic/component.js

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
-import { Tabs, TabsPanel } from 'cf-component-tabs';
+import {
+  Tabs,
+  TabsItemUnstyled,
+  TabsItemTheme,
+  TabsPanel
+} from 'cf-component-tabs';
+import { applyTheme } from 'cf-style-container';
+
+const Tab = applyTheme(TabsItemUnstyled, TabsItemTheme, () => ({
+  background: '#FFFFF0'
```

Comment:
This is not in the component at all but rather just in the example to showcase how you can override the styles of a `TabItem` component.